modules/videoio/src/cap_ffmpeg_legacy_api.hpp

```diff
@@ -30,7 +30,7 @@ OPENCV_FFMPEG_API int cvSetCaptureProperty_FFMPEG(struct CvCapture_FFMPEG* cap,
 OPENCV_FFMPEG_API double cvGetCaptureProperty_FFMPEG(struct CvCapture_FFMPEG* cap, int prop);
 OPENCV_FFMPEG_API int cvGrabFrame_FFMPEG(struct CvCapture_FFMPEG* cap);
 OPENCV_FFMPEG_API int cvRetrieveFrame_FFMPEG(struct CvCapture_FFMPEG* capture, unsigned char** data,
-                                             int* step, int* width, int* height, int* cn);
+                                             int* step, int* width, int* height, int* cn, int* depth);
```

Comment:
AFAIK, it is a public API.
We should not modify existed functions.

Reply:
I thought we've switched to the new plugin API in the FFmpeg plugin? Is it possible, what do you think?